lxd/firewall/interfaces.go

```diff
@@ -0,0 +1,32 @@
+package firewall
+
+import (
+	"net"
+
+	deviceConfig "github.com/lxc/lxd/lxd/device/config"
+)
+
+
+// Firewall represents an LXD firewall.
+type Firewall interface {
+	// Lower-level Functions
+	NetworkClear(protocol string, comment string, table string) error
+	ContainerClear(protocol string, comment string, table string) error
+	VerifyIPv6Module() error
+
+	// Proxy Functions
+	InstanceProxySetupNAT(protocol string, ipAddr net.IP, comment string, connType, address, port string, cPort string) error
+
+	// NIC Bridged Functions
+	InstanceNicBridgedRemoveFilters(m deviceConfig.Device, ipv4 net.IP, ipv6 net.IP) error
+	InstanceNicBridgedSetFilters(m deviceConfig.Device, config map[string]string, ipv4 net.IP, ipv6 net.IP, name string) error
+
+	// Network FunctionsFunctions
```

Comment:
Comment looks odd :)

Reply:
My mistake, I'll clear that up right now